docs/content/preview/admin/yb-admin.md

```diff
@@ -635,11 +633,37 @@ yb-admin \
     flush_table ysql.yugabyte table_name
 
 ```
+
 ```output
 Flushed [yugabyte.table_name] tables.
 ```
----
 
+**Syntax 2: Using table ID**
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> \
+    flush_table tableid.<table_id> [timeout_in_seconds] [ADD_INDEXES]
+```
+
+* *master_addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* table_id: The identifier (ID) of the table.
```

Comment:
"identifier" is vague.  compact table and other rules have better "UUID"

`*table_id*: The unique UUID associated with the table to be compacted.`